opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/exemplar/exemplar_reservoir.py

```diff
@@ -0,0 +1,325 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import ABC, abstractmethod
+from random import randrange
+from typing import Any, Callable, Optional, Sequence, TypeAlias, Union
```

Comment:
Looks like `TypeAlias` is not available yet in Python 3.9 (supported by this repo), but it has also been deprecated since 3.12 😅 

https://docs.python.org/3/library/typing.html#typing.TypeAlias

Reply:
nvm -- looks like you just fixed it 😄